### ISSUE TITLE
Int images are not viewable

### ISSUE THREAD

**Issue by jni** [napari/napari#308]
## 🐛 Bug

If an image has dtype `int` (int64 on most platforms), napari crashes.

## To Reproduce

Run the following code:

```python
import numpy as np
from skimage import data
import napari
from napari.util import app_context

with app_context():
    blobs = data.binary_blobs(length=128, volume_fraction=0.1, n_dim=3)
    v = napari.view(blobs=blobs)
```

Error:

```pytb
Traceback (most recent call last):
  File "image-int.py", line 8, in <module>
    v = napari.view(blobs=blobs)
  File "/home/jni/projects/napari/napari/_view.py", line 50, in view
    clim_range=clim_range, name=name)
  File "<string>", line 3, in add_image
  File "/home/jni/projects/napari/napari/components/_viewer/model.py", line 287, in add_layer
    layer.indices = self.dims.indices
  File "/home/jni/projects/napari/napari/layers/_base_layer/model.py", line 114, in indices
    self._set_view_slice()
  File "/home/jni/projects/napari/napari/layers/_image_layer/model.py", line 209, in _set_view_slice
    self.status = self.get_message(coord, value)
  File "/home/jni/projects/napari/napari/layers/_image_layer/model.py", line 378, in get_message
    msg = msg + f'{value:0.3}'
ValueError: Precision not allowed in integer format specifier
```

## Expected behavior

The image is displayed correctly.

## Environment

 - napari Version (e.g., 1.0): master
 - OS (e.g., Linux): Linux
 - Python version: 3.6
 - Any other relevant information:

**Comment by sofroniewn** [comment]
:-( looks like another problem with our string formatting, as mentioned in #93 and has been cropping up elsewhere.

@jni do you want to take a stab at fixing this across the entire codebase. It's pretty important we get it right and I think you have a good sense for the desired string outputs and the best ways to get them!!

**Comment by jni** [comment]
Yes, I'll work on this, just wanted to add it here for tracking. Another 1-2 issues incoming. =\

**Comment by sofroniewn** [comment]
this is now fixed by #422
